src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,43 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    int supply = 0;
+    int buy = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            writeToFile(toFileName, readFromFile(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
```

Comment:
what if this exception came from `writeToFile()` method?

Reply:
fixed